classroom.rb

```diff
@@ -0,0 +1,15 @@
+# Class Classroom
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(_students)
+    @add_student << student
+    student.classroom = self
+  end
```

Comment:
- [ ] Let's define the `add_student` method properly by making sure the argument passed to it is consistent and is being used inside the method. If you noticed, the argument you're passing is `_students` but there is no place that you're making use of this argument, instead, you're making use of `student` which is not defined anywhere in the method. Let's fix this by modifying the method to accept and make use of the argument and to push the argument to the `students` array defined in the `initialize` method. See my suggestion below;

```suggestion
  def add_student(student)
    @students << student
    student.classroom = self
  end
```

Reply:
Thank you for detailed comment, I fixed this